hack/verify-golint.sh

```diff
@@ -93,37 +90,19 @@ else
 		done
 		echo
 		echo 'Please fix the above errors. You can test via "golint" and commit the result.'
+		echo 'You can also add the package to hack/.golint_failures if your reviewer is okay with it.'
```

Comment:
"If the above warnings don't make sense, you can exempt this package from golint checking by adding to hack/.golint_failures (if your reviewer is okay with it)" or something like that.